website/blog/2018-06-25-on-consuming-and-publishing-es6+-packages.md

```diff
@@ -0,0 +1,218 @@
+
+- You (your supported environments)
+- Browsers (Chrome, Firefox, Edge, Safari)
+- TC39/ECMAScript Proposals (and Babel implementations)
+
+Thus the need isn't just for `6to5` to be renamed to Babel because it compiles to `7to5`, but for Babel to change the implicit assumption it only targets ES5. With `@babel/preset-env`, you are able to write the latest JavaScript and target whichever browser (environment) you want!
+
+But assuming people even know it exists or have starting using it at all, it's mainly for your **application** code.
+
+## Your Code
+
+You have control over your own code to be able to take advantage of preset-env: write in ES6+ and target ES6+ browsers.
+
+However, this isn't necessarily the case for your dependencies (though you _should_ take ownership of them).
+
+## What Then?
+
+The next step is to start thinking "how do we actually make this happen"?
+
+Is it as straightforward as just running Babel over `node_modules` with the same configuration for your application?
+
+## Caveats
+
+### Using Non-Standard Syntax
+
+This is what Dan was warning about. There are many issues with *shipping* uncompiled proposal syntax.
+
+#### Using Proposals
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="und" dir="ltr"><a href="https://t.co/femUb4vgxh">pic.twitter.com/femUb4vgxh</a></p>&mdash; Rach Smith 🌈 (@rachsmithtweets) <a href="https://twitter.com/rachsmithtweets/status/892478598765887488?ref_src=twsrc%5Etfw">August 1, 2017</a></blockquote>
+
+We already recommend that people should be careful when using proposals lower than Stage 3, let alone publishing them.
+
+But telling people not to use Stage X goes against the whole purpose of Babel in the first place. A big reason why proposals gain improvements and move forward are because of the feedback the committee gets from real-world usage (whether in production or not) based on using it via Babel.
+
+There is certainly a balance to be had here. We don't want to scare people away from using new syntax (that is a hard sell 😂), but we also don't want people to get the idea that "once it's in Babel, the syntax is official or immutable". Ideally people look into the purpose of a proposal and make the tradeoffs for their use case.
+
+One of the most common things people do is use the Stage 0 or Stage 2 presets. We plan to remove the stage presets in v7 after much back and forth. I thought at first it would be convienent, that people would make their own unofficial ones anyway, or it might help with "JavaScript fatigue". I find now that it just causes more of an issue: people continue to copy/paste configs with out understanding what goes into a preset in the first place. After all, seeing `"stage-0"` says nothing. My hope is that in making the decision to use proposal plugins explicit, people will have to learn what non-standard syntax they are opting into.
```

Comment:
This point should be highlighted more. Right now it's buried within this paragraph and is an important update for v7. It could be a paragraph heading.

> We plan to remove the stage presets in v7 after much back and forth.